test/recipes/90-test_store.t

```diff
@@ -0,0 +1,350 @@
+#! /usr/bin/env perl
+# Copyright 2015-2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use File::Spec;
+use MIME::Base64;
+use OpenSSL::Test qw(:DEFAULT srctop_file srctop_dir bldtop_file data_file);
+
+my $test_name = "test_store";
+setup($test_name);
+
+my @noexist_files =
+    ( "test/blahdiblah.pem",
+      "test/blahdibleh.der" );
+my @src_files =
+    ( "test/testx509.pem",
+      "test/testrsa.pem",
+      "test/testrsapub.pem",
+      "test/testcrl.pem",
+      "apps/server.pem" );
+my @generated_files =
+    (
+     ### generated from the source files
+
+     "testx509.der",
+     "testrsa.der",
+     "testrsapub.der",
+     "testcrl.der",
+
+     ### generated locally
+     ### These examples were pilfered from OpenConnect's test suite
```

Comment:
I found the email exchange I had with @dwmw2.  The test suite I refer to is http://git.infradead.org/users/dwmw2/openconnect.git/blob/HEAD:/tests/Makefile.am , and this is from the email that had me go ahead:

> Richard:
> > Speaking of...  do you mind if I pilfer from that Makefile.am for our
> > tests?

> @dwmw2 :
> Not at all; please do. You're welcome to use anything in the test suite under the OpenSSL licence. With the caveat/exception that bits of it came from Nikos — but that was mostly the ocserv test harness which isn't the bit you want.